pages/ksphere/kommander/latest/projects/index.md

```diff
@@ -43,3 +43,11 @@ Using project ConfigMaps and project Secrets you can define configuration resour
 ## Platform Services
 
 Platform Services provide a catalog of applications you can deploy across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information
+
+## Quotas
+
+Quotas can be set to constrain resource usage. Quotas are applied to all project clusters.
```

Comment:
Why do we capitalize Quota?

Reply:
Because we capitalize the names of project detail tabs in the other sections. For example: "Secrets", "ConfigMaps", etc...